### DISCUSSION TITLE
Tipo de evaluación en cambio de tamaños de pantalla

### DISCUSSION THREAD

**Discussion by rodrigoogalde** [IIC2513/Syllabus#15] [Tarea 1]
Hola!
En el enunciado se menciona que un 15% de la nota equivale a Responsividad y cambios de estilo asociado al tamaño de la pantalla.
Con este último, se refieren a el cambio con el zoom del navegador (Ctrl -, Ctrl +) o al display que tiene el navegador en la pantalla del PC, con esto me refiero a correr el navegador a la izquierda, derecha, parte superior, etc.
Muchas gracias!

**Comment by aaossa** [comment]
Movido a #19
